mkdocs.yaml

```diff
@@ -1,6 +1,7 @@
 site_name: vLLM
 site_url: https://docs.vllm.ai
 repo_url: https://github.com/vllm-project/vllm
+edit_uri: edit/main/docs/
```

Comment:
I think @DarkLight1337 I already used, I just did some improvements for css and open way...